### DISCUSSION TITLE
From HNews - Feedback

### DISCUSSION THREAD

**Discussion by nitsujri** [avo-hq/avo#989] [General]
Hi Adrian,

You kindly asked for feedback [from this dinosaur](https://news.ycombinator.com/item?id=31824877#31833487). I played with it for a bit and have some feedback. 

I played with it for a bit and I'll say fantastic work. You've taken a lot of ActiveAdmin's key benefits such as - one line modify controller+view and fixed a number of the problems that activeadmin has (i.e. polymorphism).

Only real feedback is information dense layouts. Easier columns and configurable whitespace.

If I was to start up a new project, I'd give this a go.

**Comment by adrianthedev** [comment]
Hey @nitsujri,

Thanks for the kind words! I love it when users understand the product.
I'm hungry for feedback so nothing is too small. 

Can you elaborate a little bit on your feedback? 

> information dense layouts

Is there too much information? On a particular view (index, show, edit) or on all?

> Easier columns

Do you mean easier to configure?

> configurable whitespace

Configurable whitespace?

**Reply by ghost** [reply]
@adrianthedev I think he means Avo has too much space (either padding or margin) and he needs a more compact design overall. A typical ActiveAdmin page usually has more information than Avo (said index page) because each row occupies less space. 

Just try to rephrase @nitsujri words. Avo is a great product after all and personally, I prefer Avo's design over classic ActiveAdmin.
